RocketChat to MS Teams updates to Developer.gov docs (#354)
bcgov/platform-developer-docs: src/docs/app-monitoring/user-defined-monitoring.md

There is a default `AlertManagerConfig` object called `platform-services-controlled-alert-routing` in each namespace that is not editable that sets out some default alerting rules. It ensures that the Products Tech Leads and Product Owner get the base level alerts.

> You can add an additional `AlertManagerConfig` to add more email contacts, or set up another notification channel like MS Teams.

Reviewer: ```suggestion
You can add an additional `AlertManagerConfig` to add more email contacts, and in future notifications channel like MS Teams may be supported.
```

> You can add an additional `AlertManagerConfig` to add more email contacts, and in future notifications channel like MS Teams may be supported.

Reviewer: ```suggestion
You can add an additional `AlertManagerConfig` to add more email contacts, and in future notifications channels like MS Teams may be supported.
```

If you want to use an email receiver, keep it the same and only update the `name` and `to` fields, as well as the `Product Name Here` in the `html`. The rest ensures that the email is sent correctly and with nice formatting.

> You can add multiple email receivers, or a webhook connector to MS Teams.

Reviewer: ```suggestion
You can add an additional `AlertManagerConfig` to add more email contacts, and in future notifications channel like MS Teams may be supported.
```

> You can add an additional `AlertManagerConfig` to add more email contacts, and in future notifications channel like MS Teams may be supported.

Reviewer: This duplicates the earlier line 213 - maybe should just be removed?
